Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/DefaultModelInfoProvider.java

```diff
@@ -81,7 +82,7 @@ public boolean accept(File path, String name) {
             if (modelFile != null) {
                 InputStream is = new FileInputStream(modelFile);
 
-                return ModelInfoXmlReader.readValue(is, ModelInfo.class);
+                return ModelInfoReaderFactory.getReader("application/xml").read(is);
```

Comment:
Wouldn't be better to define these MimeTypes somewhere?

Reply:
I am assuming this needs to be aligned with some sort of capabilities statement

Reply:
Probably, but given the standard nature, and the need to introduce a dependency to define them somewhere, I'm okay with them being magic values for now